tests/test_multipart.py

```diff
@@ -574,6 +589,23 @@ def test_read_chunk_doesnt_breaks_reader(self):
             while not part.at_eof():
                 yield from part.read_chunk(3)
 
+    def test_read_chunk_from_stream_doesnt_breaks_reader(self):
+        reader = aiohttp.multipart.MultipartReader(
+            {CONTENT_TYPE: 'multipart/related;boundary=":"'},
+            Stream(b'--:\r\n'
+                   b'\r\n'
+                   b'test'
+                   b'\r\n--:\r\n'
+                   b'\r\n'
+                   b'passed'
+                   b'\r\n--:--'))
+        while True:
+            part = yield from reader.next()
+            if part is None:
+                break
+            while not part.at_eof():
+                yield from part.read_chunk(4)
```

Comment:
Added content checks for two these tests. Chunks differs because of different chunk size, but content should be equal.